operator/pkg/securedcluster/values/translation/image_overrides.go

```diff
@@ -4,12 +4,12 @@ import "github.com/stackrox/rox/operator/pkg/images"
 
 var (
 	imageOverrides = images.Overrides{
-		images.Main:             "image.main.fullRef",
-		images.CollectorSlim:    "image.collector.slim.fullRef",
-		images.CollectorFull:    "image.collector.full.fullRef",
-		images.ScannerSlim:      "image.scanner.fullRef",
-		images.ScannerSlimDB:    "image.scannerDb.fullRef",
-		images.ScannerV4DB:      "image.scannerV4DB.fullRef",
-		images.ScannerV4Indexer: "image.scannerV4.fullRef",
+		images.Main:          "image.main.fullRef",
+		images.CollectorSlim: "image.collector.slim.fullRef",
+		images.CollectorFull: "image.collector.full.fullRef",
+		images.ScannerSlim:   "image.scanner.fullRef",
+		images.ScannerSlimDB: "image.scannerDb.fullRef",
+		images.ScannerV4DB:   "image.scannerV4DB.fullRef",
+		images.ScannerV4:     "image.scannerV4.fullRef",
```

Comment:
nit: reorder

Reply:
seen this one too late, sorry. happy to clean this up later.